experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_k8s_job.yml

```diff
@@ -0,0 +1,112 @@
+#############
+###### Note:
+#### Please ingore this file during the code review, will remove the dummy variables after the final review
+
+###########
```

Comment:
pls remove these comments, and only remove the dummy values from env,sa,image as this is the final review